01-syntax/04-high-order-functions/life-expectancy/lifeExpectancy.js

```diff
@@ -1,14 +1,123 @@
+var ANCESTRY_FILE = '[\n  ' + [
```

Comment:
The same issue here, use variable from [here](https://github.com/nesterone/sf-eng-init-gl-spring-2016/blob/master/01-syntax/04-high-order-functions/life-expectancy/index.html#L6)